_tour/operators.md

```diff
@@ -34,27 +47,62 @@ val vector3 = vector1 + vector2
 vector3.x  // 3.0
 vector3.y  // 3.0
 ```
+{% endtab %}
+{% tab 'Scala 3' for=operators_3 %}
+```scala
+case class Vec(x: Double, y: Double):
+  def +(that: Vec) = Vec(this.x + that.x, this.y + that.y)
+
+val vector1 = Vec(1.0, 1.0)
+val vector2 = Vec(2.0, 2.0)
+
+val vector3 = vector1 + vector2
+vector3.x  // 3.0
+vector3.y  // 3.0
+```
+{% endtab %}
+{% endtabs %}
+
 The class Vec has a method `+` which we used to add `vector1` and `vector2`. Using parentheses, you can build up complex expressions with readable syntax. Here is the definition of class `MyBool` which includes methods `and` and `or`:
 
+{% tabs operators_4 class=tabs-scala-version %}
+{% tab 'Scala 2' for=operators_4 %}
 ```scala mdoc
 case class MyBool(x: Boolean) {
   def and(that: MyBool): MyBool = if (x) that else this
   def or(that: MyBool): MyBool = if (x) this else that
   def negate: MyBool = MyBool(!x)
 }
 ```
+{% endtab %}
+{% tab 'Scala 3' for=operators_4 %}
+```scala
+case class MyBool(x: Boolean):
+  def and(that: MyBool): MyBool = if x then that else this
+  def or(that: MyBool): MyBool = if x then this else that
+  def negate: MyBool = MyBool(!x)
+```
+{% endtab %}
+{% endtabs %}
 
 It is now possible to use `and` and `or` as infix operators:
 
+{% tabs operators_5 %}
+{% tab 'Scala 2 and 3' for=operators_5 %}
 ```scala mdoc
 def not(x: MyBool) = x.negate
 def xor(x: MyBool, y: MyBool) = (x or y) and not(x and y)
 ```
+{% endtab %}
+{% endtabs %}
 
 This helps to make the definition of `xor` more readable.
 
 ## Precedence
 When an expression uses multiple operators, the operators are evaluated based on the priority of the first character:
+
+{% tabs operators_6 %}
+{% tab 'Scala 2 and 3' for=operators_6 %}
```

Comment:
this example isn't scala code - so I suppose remove the tabs :)

Reply:
I agree... I will update this part.